### DISCUSSION TITLE
How do I get the checkboxes for multibooking simple=0?

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#38] [Q&A] [answered]
This is the simple=1 shortcode:
`[eme_add_multibooking_form simple=1 id=62,63,64,65,66,67,68,69,70,71]`

I can get this form:
![Simple=1](https://user-images.githubusercontent.com/101639308/214549566-d35f0fae-0e29-4fc5-b2c9-8a185f01f998.png)

This is the simple=0 shortcode:
`[eme_add_multibooking_form id=62,63,64,65,66,67,68,69,70,71 template_id_header=22 template_id_footer=23]`

As I need more info from the participants, simple=1 does not work for me. But when I use simple=0, the checkboxes go missing:
![Simple=0](https://user-images.githubusercontent.com/101639308/214550192-ce7a822c-eaad-407b-bb22-906f739e77bf.png)

Also, if I put in the body template as stated in the documentation:
`<li>`
`<strong>#_{d M Y (l)} #_24HSTARTTIME - #_24HENDTIME</strong>#_SEATS<br/>`
`<br /><br />`
`</li>`

The checkboxes and event names go missing. I know I need to add a placeholder to show the event names. But after searching the "Registration form formatting" documentation, I can't find anything that shows event names:
![Body template](https://user-images.githubusercontent.com/101639308/214550544-d46ab03b-cb24-415c-bc36-169009bbcb8c.png)

Finally, if I use the recurrence id in the shortcode, where 13 is virtual and 14 is in-person:
`[eme_add_multibooking_form recurrence_id=13,14 template_id_header=22 template_id_footer=23]`

The entire in-person info goes missing and looks like this:
![Recurrence id](https://user-images.githubusercontent.com/101639308/214552595-44946711-a91b-4e13-8f91-c0460b831139.png)

How do I get the simple=0 form to look like the one in simple=1 with the checkboxes? Is there a simpler way to use the recurrence id instead of listing all the individual events?

**Comment by liedekef** [comment, marked as answer]
First: for all event placeholders, see the doc on event placeholders: https://www.e-dynamics.be/wordpress/category/documentation/7-placeholders/7-2-events/ . As you can see, #_EVENTNAME is one of them.

Second: the doc was a bit wrong in the default (the default of `#_EVENTNAME #_STARTDATE #_STARTTIME: #_SEATS <br />` is taken if no body template is used in case of simple=0. This has been corrected.

Third: the difference between simple=0 and simple=1 is exactly the select boxes in case min and max allowed seats to book=1 (which seems to be the case based on your screenshots):
- in the case of simple=0 (the default) and minx/max seats to book=1, #_SEATS renders an invisible field. Meaning: people will attend all the mentioned event upon rsvp, if you don't want that you'd need to set the min seats to book=0 then.
- in the case of simple=1 (the default) and minx/max seats to book=1, the event list then becomes a simple select list.

Fourth: asking for extra info in the multibooking form: can be done by adding custom fields to the footer template (or header), like with a regular event.

Fifth: the recurrence_id option only allows 1 recurrence id, not multiple. If you want events from multiple recurrences etc ..., put all those events in a category and use the category_id option.

**Comment by refugeindhamma** [comment]
Thanks for your prompt reply!

As a number of my participants are seniors, I would like to make it as easy for them as possible. So, is it possible to use select list radio buttons instead of selecting 1 seat from the dropdown menu when using simple=0?

**Reply by liedekef** [reply]
What's wrong with using simple=1 then?

**Reply by liedekef** [reply]
Also: when using simple=0 with min-seats-to-book=0 and max=1, you can force a select box if you make the event "attendance only".

**Reply by refugeindhamma** [reply]
> Also: when using simple=0 with min-seats-to-book=0 and max=1, you can force a select box if you make the event "attendance only".

This is great! Maybe you can put this as a "Tip & Trick" of EME somewhere in your documentation. Thanks for your help.

**Reply by liedekef** [reply]
Well, it is not a select box but a checkbox :-) Also, the next version of EME will clarify that in the RSVP options:
`If this option is set and the setting "Min number of seats to book" is set to 0, then the field to choose the number of seats to book will be turned into a checkbox.`
